jobs/build-snaps/build-release-cdk-addons.groovy

```diff
@@ -158,31 +164,27 @@ pipeline {
                                 break
                             fi
                         done
-                        for multi in \${MULTI_ARCH_IMAGES}
-                        do
-                            if echo \${RAW_IMAGE} | grep -qi \${multi}
-                            then
-                                # inject '-arch:' between the image name and version, as our templates expect
-                                RAW_IMAGE=\${RAW_IMAGE%%:*}-${params.arch}:\${RAW_IMAGE#*:}
-                                break
-                            fi
-                        done
 
                         # Tag and push
-                        docker tag \${i} \${TAG_PREFIX}/\${RAW_IMAGE}
+                        sudo lxc exec image-processor -- ctr image tag \${i} \${TAG_PREFIX}/\${RAW_IMAGE}
```

Comment:
Whats the format here for adding the registry, I couldnt find anything in their source code or documentation

Reply:
We can, but I don't think we need to because the tag will contain the URL to the registry, so should just be a case of pushing with the creds (`--user"${USERNAME}:${PASSWORD}"`).  Or am I missing something?

Reply:
Ah you're right I missed the tag_prefix had the URL in there.  With that said I think this was all we needed then?

Reply:
I think so too